roles/cinder-common/tasks/main.yml

```diff
@@ -35,7 +35,13 @@
     - /var/lib/cinder/lock
 
 - name: cinder log dir
-  file: dest=/var/log/cinder state=directory mode=0755 owner=cinder group=cinder
+  file: dest=/var/log/cinder state=directory mode=2750 owner=cinder group=adm
+
+- acl: name=/var/log/cinder state=present entry={{ item }}
```

Comment:
This task needs a name. Every place we run into a task without a name please add a name.